test/parallel/test-require-json.js

```diff
@@ -24,10 +24,6 @@ require('../common');
 const assert = require('assert');
 const fixtures = require('../common/fixtures');
 
-try {
-  require(fixtures.path('invalid.json'));
-} catch (err) {
-  assert.ok(
-    /test[/\\]fixtures[/\\]invalid\.json: /.test(err.message),
-    `require() json error should include path: ${err.message}`);
-}
+assert.throws(function() {
+  require(fixtures.path('invalid.json'))
```

Comment:
Pretty sure this line will result in a lint error due to the absence of a semicolon. Run `make lint-js` (or `vcbuild lint-js` on Windows) to run the linter. (It should automatically run if you do `make test`.)

```suggestion
  require(fixtures.path('invalid.json'));
```